src/models/components.d.ts

```diff
@@ -6,6 +6,37 @@ export interface SearchResultItemType {
   image: string;
 }
 
+export interface filterItem {
+  name: string;
+  amount: number;
+}
+
+export interface ageItem {
+  start: number;
+  end: number;
+  unit: string;
+}
+
+export type lifeStageItem = {
+  name: string;
+  icon: SvgIconComponent;
+};
+
+export interface content {
```

Comment:
Your content interface should have the following:
1. nutrientName
2. nutrientAmount
3. nutrientUnit
3. nutrientDVPercent
4. rdi which is a combination of `amount` and `unit`

Currently, you have combined nutrient name, amount and unit as a `string`. This is not correct. Your component will get this data separately in the JSON. 

For example `{"nutrientName": "carbohydrates",`
   `"nutrientAmount": "2",`
   `"nutrientAmountUnit": "mg"}`

Please refactor the code accordingly.

Reply:
In the other task assigned by @hhimanshu, I created and updated the JSON file for this. Should I use the @hhimanshu recommended by him?

Reply:
done